.github/workflows/release-manifests.yml

```diff
@@ -0,0 +1,35 @@
+name: Release Manifest
+
+# This workflow should be run after the integration is released into a new infrastructure-bundle image version
+# or just want to bump the bundle version for the same integration version.
+
+on:
+  workflow_dispatch:
+    inputs:
+      bundleVersion:
+        description: 'Version of Infra Bundle image without `v` prefix e.g. 2.6.4'
+        required: true
+      releaseVersion:
+        description: 'Version of nri-ecs without `v` prefix e.g. 1.3.1'
+        required: true
+
+jobs:
+  release-manifest:
+    needs: [ release-packages ]
+    name: Release manifest
+    runs-on: ubuntu-latest
+    env:
+      S3_BASE_FOLDER: s3://nr-downloads-main/infrastructure_agent
+    steps:
+      - uses: actions/checkout@v2
+      - if: ${{ github.event.release.prerelease }}
+        run: |
+          echo "S3_BASE_FOLDER=$S3_BASE_FOLDER/test" >> $GITHUB_ENV
+      - uses: aws-actions/configure-aws-credentials@v1
+        with:
+          aws-access-key-id: ${{ secrets.COREINT_AWS_ACCESS_KEY_ID }}
+          aws-secret-access-key: ${{ secrets.COREINT_AWS_SECRET_ACCESS_KEY }}
+          aws-region: us-east-1
+      - name: Upload configuration files to https://download.newrelic.com
+        run: |
+          make upload_manifests RELEASE_VERSION=${{ github.event.inputs.releaseVersion }} INFRA_BUNDLE_VERSION=${{ github.event.inputs.bundleVersion }}
```

Comment:
>:(

Reply:
![Screen Shot 2021-08-04 at 9 59 39 AM](https://user-images.githubusercontent.com/43335750/128144309-b9d8e74d-31ee-4fc8-8370-e5831b5c4e8c.png)

Reply:
I have removed all new lines from steps already to be consistent with the resto of workflows.

Reply:
oh i just understood that you where refering to the EOF line 🤣  , i missed that and merged